packages/apps/esm-login-app/src/config-schema.ts

```diff
@@ -68,6 +68,29 @@ export const configSchema = {
       _description: 'The alternative text for the logo image, displayed when the image cannot be loaded or on hover.',
     },
   },
+  footer: {
+    logos: {
```

Comment:
```suggestion
    additionalLogos: {
```

Also modify the type element and the mock